pkg/api/types.go

```diff
@@ -459,6 +459,10 @@ type PersistentVolumeSpec struct {
 	// simply fail if one is invalid.
 	// +optional
 	MountOptions []string
+	// volumeMode defines if a volume is intended to be used with a formatted filesystem
+	// or to remain in raw block state.
+	// +optional
```

Comment:
There should be a note that this field is alpha.
There should be a note that "Filesystem" is implied when no mode is specified (word "default" is not appropriate here as there is no defaulting..."